### ISSUE TITLE
databrickslabs-jupyterlab configuration breaks for conda 4.8.0

### ISSUE THREAD

**Issue by aditya-chaturvedi** [databrickslabs/jupyterlab-integration#2]
conda version is latest i.e. greater than 4.7.5 - still databrickslabs-jupyterlab complains of it being too old.
```
(dbconnect) ~ λ databrickslabs-jupyterlab $PROFILE -s -i $CLUSTER_ID
Too old conda version:
Please update conda to at least 4.7.5
(dbconnect) ~ λ conda --version
conda 4.8.0
```

**Comment by aditya-chaturvedi** [comment]
I think this line has a clue to this behavior - 

[https://github.com/databrickslabs/jupyterlab-integration/blob/44262c39cfc83956b7fcc8a5b47282c30fbf28c5/databrickslabs-jupyterlab#L162](url)

changing from 4.7.**5** to 4.8.**0** might've broken it.

**Comment by bernhard-42** [comment]
@aditya-chaturvedi Thanks for both reporting the issue and coming up with a pull request!
I am planning to come up with a new release this week - including this fix

**Comment by bernhard-42** [comment]
fyi, @aditya-chaturvedi, I have published a new version (1.0.9) including your fix and some updates to python package versions, see https://github.com/databrickslabs/jupyterlab-integration/blob/master/Changes.md
